scribble-lib/scribble/base.rkt

```diff
@@ -44,7 +44,7 @@
 
 (define (gen-tag content)
   (datum-intern-literal
-   (regexp-replace* #px"[^-a-zA-Z0-9_=\u4e00-\u9fa5\u3040-\u309F\u30A0-\u30FF]" (content->string content) "_")))
+   (regexp-replace* "[^-a-zA-Z0-9_=\u4e00-\u9fa5\u3040-\u309F\u30A0-\u30FF]" (content->string content) "_")))
```

Comment:
Keeping `#px` is better, because it means that the regexp will be compiled just once, instead of every time that `regexp-replace` is called.